docs/sphinx/reference-frontend-api.rst

```diff
@@ -798,3 +798,21 @@ Functions
 .. function:: bool obs_frontend_is_theme_dark(void)
 
    :return: Checks if the current theme is dark or light.
+
+---------------------------------------
+
+.. function:: const char *obs_frontend_get_last_recording(void)
+
+   :return: Gets the last recording.
```

Comment:
Just a nit, I think this should be "The file path of the last recording", and similarly for the other two functions. The :returns: should not need to say "Gets". Also, explicitly saying "file path" is clearer, especially for the screenshot, which someone might interpret as the raw image data. Of course, the return type is a const char *, but imo it's better to spend a few more characters for clarity.

Reply:
Fixed.